CHANGELOG.md

```diff
@@ -32,6 +38,8 @@
 - Fix off-by-one error in `kafka_consumer` committing offsets, thus replaying the last committed event.
 - Allow `kafka_consumer` connector to reconnect upon more error conditions and avoid stalls.
 - Include flow alias in pipeline and connector aliases reported via metrics events and logging in order to deduplicate entries
+- Add clear error messages for pipelines for console in, pipeline out, pipeline in
```

Comment:
should this be in the `0.13.0-rc.1` section? Or should the above be down here as well?